src/createServer.js

```diff
@@ -1,3 +1,49 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+
+const { getUrlParameters } = require('./URLParameters');
+const { checkURL } = require('./catchErrors');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const [text, query] = req.url.split('?');
```

Comment:
Use the 'url' module to properly parse the URL and query parameters.

Reply:
Fixed